### PR TITLE
Fix code compatibility issue after catching up latest openocd master branch

### PR DESCRIPTION
**Pull request by chernyee-rs**

### CODE REVIEW

File: tcl/board/gemini.cfg [hunk 1]
```diff
@@ -20,7 +20,7 @@ jtag newtap $_CHIPNAME tap -irlen 5 -expected-id 0x1000563d
 target create $_TARGETNAME riscv -endian little -chain-position $_CHIPNAME.tap
 
 # Gemini PLD/FPGA configuration
-pld device gemini $_TARGETNAME
+pld create gemini.pld gemini $_TARGETNAME
```
Comment (guanyung-rs) [inline tcl/board/gemini.cfg:23]: We will need update the fpga configuration command in the programmer tool

**Review by guanyung-rs — APPROVED** [review]
If the test passed, then go ahead to merge it.

**Review by esther-rs — APPROVED** [review]

File: tcl/board/gemini.cfg (hunk 1 above)
Comment (chernyee-rs) [reply tcl/board/gemini.cfg:23]: Yes.

**Comment by chernyee-rs** [comment]
Tested on virgo prototype and the catch up code work fine.

**Pull request merged**